package.json

```diff
@@ -74,6 +74,7 @@
         "lodash": "^4.17.15",
         "moment": "2.24.0",
         "node-forge": "^0.10.0",
+        "rc-tree": "^4.0.0-beta.2",
```

Comment:
Don't we have a stable version that has the required features. Maybe the last `3.x.x` version?
If not shall we create an issue to keep track to bump the version when the `4.0.0` stable version is released.

Reply:
Also shall we install `https://www.npmjs.com/package/@types/rc-tree` to get typings support?

Reply:
@jeradrutnam and I had a discussion regarding this offline and this issue https://github.com/wso2/product-is/issues/10599 will be used to track the version bump.